projects/Mallard/src/authentication/AccessContext.tsx

```diff
@@ -115,7 +116,7 @@ const AccessProvider = ({
     const value = useMemo(
         () => ({
             attempt,
-            canAccess: !!attempt && isValid(attempt),
+            canAccess: (!!attempt && isValid(attempt)) || isNotRun(attempt),
```

Comment:
what does "isNotRun" mean exactly? It doesn't mean that people would be able to access things without any attempt to log-in being run I'm guessing, so I'm wondering whether there could be a better name.